tests/devices/test_default_qubit_torch.py

```diff
@@ -20,10 +20,18 @@
 import pytest
 import cmath
 import math
+import functools
 
 torch = pytest.importorskip("torch", minversion="1.8.1")
 
+torch_devices = [None]
```

Comment:
If we don't want to use only a `gpu` folder for gpu tests, I would recommend adding a `gpu` mark to our pytest suite and marking this entire file with 
```
@pytest.mark.gpu
```
We would then need to add `gpu` to `tests/pytest.ini` like `slow`.

Reply:
Thanks! Added the `gpu` marker to the 3 test files contained in the PR description.